contrib/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# example of Dockerfile that builds release of electrumx-1.13.0
+# ENV variables can be overrided on the `docker run` command
+
+FROM ubuntu:18.04
+
+WORKDIR /
+ADD https://github.com/kyuupichan/electrumx/archive/1.13.0.tar.gz /
```

Comment:
🤔 Could we change this to something like git clone https://github.com/kyuupichan/electrumx?

Reply:
Please do. I was just pasting a Dockerfile that works for me.